docs/quick_start.rst

```diff
@@ -226,179 +367,245 @@ A user-supplied parsing function is employed to parse the output files.
+  ----------                        -------    --------   --------             -----
+  bubble-sort                                  4/4                             
+  insertion-sort                               4/4                             
+  merge-sort ~ bba-insertion, bs2              4/4                             
+  merge-sort ~ bba-insertion, bs20             4/4                             
+  merge-sort ~ bba-insertion, bs200            4/4                             
+  merge-sort ~ bba-selection, bs2              4/4                             
+  merge-sort ~ bba-selection, bs20             4/4                             
+  merge-sort ~ bba-selection, bs200            4/4                             
+  32 experiments in total
 
 
-To enable automated builds, we need to add ``builds`` and ``revisions``
-stanzas to ``experiments.yml``. In our example, these look like:
+Builds and Revisions
+--------------------
 
-.. code-block:: YAML
+To make sure that experiments always run using the same program binaries,
+simexpal can manage internal projects as well as external git repositories,
+automatizing the build process.
 
-    builds:
-      - name: simexpal
-        git: 'https://github.com/hu-macsy/simexpal'
-        configure:
-          - args:
-            - 'cmake'
-            - '-DCMAKE_INSTALL_PREFIX=@THIS_PREFIX_DIR@'
-            - '@THIS_CLONE_DIR@/examples/sorting_cpp/'
-        compile:
-          - args:
-            - 'make'
-            - '-j@PARALLELISM@'
-        install:
-          - args:
-            - 'make'
-            - 'install'
-
-    revisions:
-      - name: main
-        build_version:
-          'simexpal': 'd8d421e3c2eaa32311a6c678b15e9e22ea0d8eac'      # specify the SHA-1 hash of a tagged commit (recommended)
-                                                                      # it is also possible to checkout the top commit
-                                                                      # of a branch by specifying the SHA-1 hash or
-                                                                      # branch name (not recommended for reproducibility reasons)
-
-Next, we have to assign the builds to their respective experiments:
+Automated builds are controlled by the ``builds`` and ``revisions`` stanzas
+in the ``experiments.yml``.
 
-.. code-block:: YAML
+For the remainder of this section, we will will use the `C++ implementation
+<https://github.com/hu-macsy/simexpal/tree/master/examples/sorting_cpp>`_ of the
+sorting example. We will use simexpal to resolve the dependency and to configure
+and compile the C++ project. 
 
-    experiments:
-      - name: quick-sort
-        use_builds: [simexpal]  # specify which builds get used for this experiment
-        args: ['quicksort', '@INSTANCE@', '@EXTRA_ARGS@']
-        stdout: out
+simexpal will invokes CMake commands to build the program; these steps are
```

Comment:
There is an "s" too much: "simexpal will invoke CMake commands"

Reply:
Why?

Reply:
I phrased that poorly, there is an "s" too much in "invokes" and it should be "simexpal will invoke CMake commands ..." (or "simexpal envokes CMake commands ..."

Reply:
Ahh! Thanks! Done.